defaultLibraries/kliopt/src/main/kotlin/org/jetbrains/kliopt/ExperimentalCli.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+
+package org.jetbrains.kliopt
+
+import kotlin.annotation.AnnotationTarget.*
+
+/**
+ * This annotation marks the experimental API for working with command line arguments.
+ *
+ * > Beware using the annotated API especially if you're developing a library, since your library might become binary incompatible
+ * with the future versions of the CLI library.
+ *
+ * Any usage of a declaration annotated with `@ExperimentalCli` must be accepted either by
+ * annotating that usage with the [UseExperimental] annotation, e.g. `@UseExperimental(ExperimentalCli::class)`,
+ * or by using the compiler argument `-Xuse-experimental=org.jetbrains.kliopt.ExperimentalCli`.
+ */
+@Experimental(level = Experimental.Level.ERROR)
```

Comment:
Why so harsh?

Reply:
Set another level? I made same wsy as is done for stdlib in example which @ilya-g sent.